OrbitGl/TimeGraph.cpp

```diff
@@ -100,35 +100,35 @@ void TimeGraph::Clear() {
 double GNumHistorySeconds = 2.f;
 
 //-----------------------------------------------------------------------------
-bool TimeGraph::UpdateSessionMinMaxCounter() {
-  m_SessionMinCounter = LLONG_MAX;
+bool TimeGraph::UpdateCaptureMinMaxCounter() {
+  m_CaptureMinCounter = LLONG_MAX;
```

Comment:
We should probably replace occurences of LLONG_MAX by std::numeric_limits<uint64_t>::max();  Doesn't need to be in this PR.

Reply:
Done.